pynxtools/nexus/nexus.py

```diff
@@ -776,7 +776,7 @@ def process_nexus_master_file(self, parser):
 )
 def main(nexus_file, documentation, concept):
     """The main function to call when used as a script."""
-    logging_format = "%(levelname)s: %(message)s"
+    logging_format = "%(message)s"
```

Comment:
Should not this change require the modification of the reference files for the nexus tests?